src/main/java/com/edmunds/rest/databricks/DTO/scim/user/AddUsersToGroupOperation.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Edmunds.com, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.edmunds.rest.databricks.DTO.scim.user;
+
+import com.edmunds.rest.databricks.DTO.scim.Operation;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class AddUsersToGroupOperation extends Operation {
+
+  private long[] members;
+
+  public AddUsersToGroupOperation(long[] members) {
+    this.members = members;
+    op = "add";
+
+    HashMap<String, Object> membersMap = new HashMap<>();
+    List<HashMap<String, String>> values = new ArrayList<>(members.length);
```

Comment:
Can you use a [multimap](https://guava.dev/releases/19.0/api/docs/com/google/common/collect/Multimap.html) here?

Reply:
The code looks a little bit cleaner with a multimap but AddUsersToGroupOperation will need it's own custom jsonserializer. It's not hard but I'm not sure what's the best option - we lose complexity on one side and gain on another one. @javamonkey79 @samshuster  @Mikhail-Ivanov wdyt?

Reply:
I am personally ok with keeping it as is, but will let @javamonkey79 make the final call.

Reply:
It looks like you can convert it to a map during serialization, see [here](https://stackoverflow.com/a/25676798). If you can't get that to work, then just keep it as is.

Reply:
@cornelcreanga this comment won't block approval, I'd just like to know your opinion, before moving on. wdyt?